src/redux/actions/signupAction.js

```diff
@@ -0,0 +1,31 @@
+import { toast } from 'react-toastify';
+import * as actions from './actionTypes';
+
+export const succesfulSignUpAction = (data) => ({
+  type: actions.SIGNUP_SUCCESS,
+  payload: data,
+});
+
+export const failedSignUpAction = (error) => ({
+  type: actions.SIGNUP_ERROR,
+  payload: error,
+});
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
solve this eslint error, I think you have to export something as a default instead of disabling the rule.

Reply:
fixed